pymc/aesaraf.py

```diff
@@ -198,22 +170,40 @@ def extract_obs_data(x: TensorVariable) -> np.ndarray:
     raise TypeError(f"Data cannot be extracted from {x}")
 
 
+def extract_rv_and_value_vars(
+    random_var: TensorVariable,
+) -> Tuple[TensorVariable, TensorVariable]:
```

Comment:
The type hint is wrong here. It should say that the `TensorVariable`s are `Optional`.